backend/editor/github_functions.py

```diff
@@ -104,6 +105,7 @@ async def update_file(self, filename: str, file_sha: str) -> FileCommit:
                     content=base64.b64encode(new_file_contents.encode("utf-8")),
                     sha=file_sha,
                     branch=self.branch_name,
+                    committer=author,
```

Comment:
I think it's better to use author=author

Think of git being used initially by exchanging patches through emails. Author is the author of the changes, Commiter is the reviewer who, after review, includes them in the git tree. 

PS: can you add a link to https://docs.github.com/en/rest/repos/contents?apiVersion=2022-11-28#create-or-update-file-contents as a comment above this function call ? (because githubkit does not really document as it auto-generate functions, and I think it's useful to be able to understand what the function does…)